libseccomp-sys/Cargo.toml

```diff
@@ -8,3 +8,12 @@ repository = "https://github.com/libseccomp-rs/libseccomp-rs"
 categories = ["seccomp", "api-bindings"]
 edition = "2018"
 readme = "README.md"
+
+[features]
+# Allows downloading and building libseccomp at build time.
+bundled = ["cc"]
+
+[build-dependencies]
+# Used to get compiler information to set CC and CFLAGS env variables when
+# compiling bundled libseccomp.
+cc = { version = "^1", optional = true }
```

Comment:
>Because we're not using the cc crate to actually compile libseccomp, only to get cflags matching the cargo profile (i.e. --debug vs --release), I don't think using the parallel feature will make a difference here. I checked and it does not change the contents of the cflags.

Oh, I misunderstood, thank you for the explanation. 

 > Per the information in [the environment variables documentation for NUM_JOBS](https://doc.rust-lang.org/cargo/reference/environment-variables.html), I was able to set the MAKEFLAGS environment variable to the value of CARGO_MAKEFLAGS, which includes the -j option, and it did reduce the compile time when testing locally.
I'll include this change in the next revision.

Greate, it would be better to set the make env variable.